### PR TITLE
multi-point constraints

### PR DESCRIPTION
**Pull request by rc**

### CODE REVIEW

File: sfepy/discrete/fem/lcbc_operators.py [hunk 1]
```diff
@@ -141,6 +141,72 @@ def __init__(self, name, regions, dof_names, dof_map_fun,
...
+            mtx_r = nm.zeros((dim * n_nod, dim), dtype=nm.float64)
+            mtx_r[0::dim,1] = coors[:,2]
+            mtx_r[0::dim,2] = -coors[:,1]
+            mtx_r[1::dim,0] = -coors[:,2]
+            mtx_r[1::dim,2] = coors[:,0]
+            mtx_r[2::dim,0] = coors[:,1]
+            mtx_r[2::dim,1] = -coors[:,0]
+
+        else:
+            msg = 'dimension in [2, 3]: %d' % dim
```
Comment (VascoSch92) [inline sfepy/discrete/fem/lcbc_operators.py:197]: Perhaps a better error message?

`Expected dimension 2 or 3 but got ...`

Comment (rc) [reply sfepy/discrete/fem/lcbc_operators.py]: Thanks, fixed. Note that it is still a work in progress, so further changes are to be expected.

**Comment by rc** [comment]
Hey @vlukes, can you try the updated example?

**Comment by vlukes** [comment]
> Hey @vlukes, can you try the updated example?

Yes, it works as expected.

**Pull request merged**